modules/wowza/main.tf

```diff
@@ -105,7 +119,7 @@ resource "azurerm_private_dns_zone_virtual_network_link" "vnet_link" {
 }
 
 resource "azurerm_private_dns_a_record" "sa_a_record" {
```

Comment:
this needs to change at some point you should not be using your own private dns zone, you need to use the central ones

Reply:
Added ticket to resolve this. thank you